docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,193 +1,172 @@
 ---
-title: Getting Started With Access Controls
-description: Get started using Teleport Access Controls.
+title: Getting Started With Role-Based Access Control
+description: Teleport Role-Based Access Control.
 ---
 
-In Teleport, any local, SSO, or robot user can be assigned one or several roles.
-Roles govern access to databases, SSH servers, Kubernetes clusters, Windows
-desktops, and web apps.
+In Teleport, RBAC (Role-Based Access Control) is a system that helps you manage who can access various resources such as databases, SSH servers, Kubernetes clusters, Windows desktops, and web apps. 
 
-We will start with local users and preset roles, assign roles to SSO users, and
-wrap up with creating your own role.
+## What is RBAC in Teleport?
```

Comment:
```suggestion
```
The first couple of sections are very short, so I think we can consolidate them.